services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/resource_tracker_container_service.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime, timedelta, timezone
+from typing import Annotated
+
+from fastapi import Depends
+from models_library.products import ProductName
+from models_library.resource_tracker import ContainerListAPI, ContainerStatus
+from models_library.users import UserID
+from pydantic import PositiveInt
+
+from ..api.dependencies import get_repository
+from ..models.pagination import LimitOffsetParamsWithDefault
+from ..models.resource_tracker_container import ContainerListDB
+from ..modules.db.repositories.resource_tracker import ResourceTrackerRepository
+
+_MAGIC_NUMBER = 3.5  # We will need to store pricing in the DB
```

Comment:
suggestion: maybe call that something more appropriate that would not need comments?
osparcum, _OSPARC_TOKEN_PRICE...

Reply:
changed: _OSPARC_TOKEN_PRICE -> but I would still leave there the comments until we remove it and take it from the database